pkg/utils/bpf-iter-ns/bpf-iter-ns.go

```diff
@@ -96,19 +96,19 @@ func ReadOnHostPidNs(iter *link.Iter) ([]byte, error) {
 		return nil, fmt.Errorf("empty /proc/self symlink")
 	}
 
-	// Create a temporary directory in the host bpffs
+	// Create a temporary directory in bpffs
 	bpfFS := "/sys/fs/bpf"
-	tmpPinDir, err := os.MkdirTemp(host.HostRoot+bpfFS, "ig-iter-")
+	tmpPinDir, err := os.MkdirTemp(bpfFS, "ig-iter-")
 	if err != nil {
 		return nil, fmt.Errorf("creating temporary directory in bpffs: %w", err)
 	}
 	defer os.RemoveAll(tmpPinDir)
 
 	// Prepare the pin path from the container and host point of view
-	pinPath := filepath.Join(tmpPinDir, "iter")
-	pinPathHost := filepath.Join(bpfFS, filepath.Base(tmpPinDir), "iter")
+	pinPathFromContainer := filepath.Join(tmpPinDir, "iter")
+	pinPathFromHost := filepath.Join("/proc", selfPidOnHost, "root", pinPathFromContainer)
```

Comment:
What's inside `selfPidOnHost`, the container's PID from the host POV?

Reply:
Yes, exactly.